scripts/xtensa-build-all.sh

```diff
@@ -86,14 +86,15 @@ then
 	exit 1
 fi
 
-if [ "x$USE_PRIVATE_KEY" == "xyes" ]
+if [ -n "${PRIVATE_KEY}" ]
 then
-	if [ -z ${RIMAGE_PRIVATE_KEY+x} ]
+	if [ -f "${PRIVATE_KEY}" ]
```

Comment:
The commit message doesn't match the code change: this is not "fixing" the feature, this is changing its user interface (which may break existing usages). The existing code may have a less pleasant interface but it seems to work:
```
$ RIMAGE_PRIVATE_KEY=fubar.key ./scripts/xtensa-build-all.sh -k apl
...
+ cmake -DTOOLCHAIN=xtensa-apl-elf -DROOT_DIR=SOF/sof/../xtensa-root/xtensa-apl-elf -DRIMAGE_PRIVATE_KEY=fubar.key ..
```
I think the real reason is that a later commit needs this; in that case just state this in the commit message.